input/fsh/SD_RadiotherapyDiseaseStatus.fsh

```diff
@@ -37,13 +37,13 @@ Description: "Extension providing a qualifier for a disease progression"
 * value[x] 1..1
 
 
-Instance: cancer-disease-status-22A
+Instance: cancer-disease-status-22B
 InstanceOf: RadiotherapyDiseaseStatus
```

Comment:
Can we move this and any other examples out of the SD_.fsh files?

Reply:
Moved the disease status example to a separate file and did following changes to the example:
- partOf references a Course Summary example. Not sure if that will be the main way this is linked, but sounds correct that this observation was done during the Course of Treatment (Procedure).
- changed patient reference to match the referenced Course Summary. Referenced to diagnosis, practitioner, and volumes were already matching this example.
- set a date that is close to treatment start.